cmd/go/main.go

```diff
@@ -0,0 +1,48 @@
+package main
+
+import (
+	"fmt"
+	"log"
+	"strings"
+
+	"github.com/kellegous/go/backend"
```

Comment:
Not sure if I follow this much in this project, but we should group the imports with standard lib in the first group, third party packages in the 2nd group and local packages in the final group.

```
import (
    "fmt"
    "log"
    "strings"

    "github.com/spf13/pflag"
    "github.com/spf13/viper"

    "github.com/kellegous/go/backend"
    "github.com/kellegous/go/backend/leveldb"
    "github.com/kellegous/go/web"
)
```

Reply:
fixed in `main.go` and in `web/web.go`, the only two places I found it out of order.